### PR TITLE
push node_list to mongodb for analytics

### PR DESCRIPTION
**Pull request by andre-merzky**
Push the node list to MongoDB, so that Analytics can evaluate the amount of available resources (the cu slots otherwise only document the amount of *used* resources).

### CODE REVIEW

File: src/radical/pilot/agent/bootstrap_0.sh [hunk 1]
```diff
@@ -1527,7 +1527,7 @@ PB1_LDLB="$LD_LIBRARY_PATH"
 #        We should split the parsing and the execution of those.
 #        "bootstrap start" is here so that $PILOT_ID is known.
 # Create header for profile log
-if ! test -z "$RADICAL_PILOT_PROFILE"
+if ! test -z "$RADICAL_PILOT_PROFILE$RADICAL_PROFILE"
```
Comment (iparask) [inline src/radical/pilot/agent/bootstrap_0.sh]: Why are you checking twice for the environment variable?

Comment (andre-merzky) [reply src/radical/pilot/agent/bootstrap_0.sh]: This checks if either `RADICAL_PILOT_PROFILE` or `RADICAL_PROFILE` is set.  Both are valid by now (per RU changes), so either one should trigger profiling.

**Comment by mturilli** [comment]
Is this ready to be merged?

**Comment by andre-merzky** [comment]
This supports the prelim proposal data, but otherwise is not urgent.   We can probably put it into the next release.

**Pull request merged**